nodeoperators/nodelist.json

```diff
@@ -2881,18 +2881,42 @@
     },
     {
         "Role": "access",
-        "Address": "192.168.56.114:3569",
```

Comment:
Bithum could not run their access nodes on mainnet-8 since they were not using a public IP. This time they have provided me with a new access node id which has a public IP and asked to replace the existing two access nodes with this new one.